app/grandchallenge/challenges/tasks.py

```diff
@@ -116,3 +124,72 @@ def save_phase():
                 )
 
             save_phase()
+
+
+@transaction.atomic
+@acks_late_micro_short_task
```

Comment:
Unfortunately there is a dependency on the order:

```suggestion
@acks_late_micro_short_task
@transaction.atomic
```